docs/guides/elements-lookup-troubleshooting.md

```diff
@@ -146,3 +146,13 @@ The final recommendation there would be:
   to improve the corresponding WebDriverAgent sources, but keep in mind there are many automation
   tests around that rely on the current way these attributes are calculated, and we probably don't
   want to break them.
+
+## Symptom #4
+
+The desired element is shown in the page tree, but its property value contains partial value. e.g. if the element's `value` attribute had 600 bytes words, the XML page tree includes only 512 bytes. Rest of them are cut off.
+
+## Resolutions To Symptom #4
+
+Get the element attribute after getting the element. For example, getting the element with `findElement` and getting the `value` from the element attribute. The individual attribute gets the element attribute by getting snapshot deeply to return the entire value. Please see [this issue](https://github.com/appium/appium-xcuitest-driver/issues/2552) and [this PR](https://github.com/appium/WebDriverAgent/pull/1007) about the details.
+
+Apple's XCTest framework might include partial text for long wording as the default snapshot. When you inspect such an element, which has long text, with [debugDescription](https://developer.apple.com/documentation/xctest/xcuielement/1500909-debugdescription) in XCTest for UI (not Appium XCUITest driver), it prints the value partially while the [value](https://developer.apple.com/documentation/xctest/xcuielementattributes/value) attribute like `element.value` prints entire the value. It looks like Apple designs the XCTest framework so.
```

Comment:
Apple's XCTest framework might include partial text for long wording as the default snapshot -> XCTest framework cuts off long element values in snapshots to achieve the best performance and to reduce the memory footprint.

Reply:
When you inspect such an element,  -> When such element is inspected ...

Reply:
It looks like Apple designs the XCTest framework so. -> this sentence is not needed

Reply:
prints entire the value -> prints the entire value